### PR TITLE
Change eyedropper icon to actual eyedropper and change cursor to cross hair.

### PR DESCRIPTION
**Pull request by xp4xbox**
Fixes: #5339 

This PR fixes the issues mentioned in 5339.

NOTE: Ignore the first commit.

### CODE REVIEW

File: resources/g2/sprites.json [hunk 1]
```diff
@@ -434,5 +434,10 @@
         "path": "edge/skyscraper_b.png",
         "x_offset": 2,
         "y_offset": 2
+    },
+    {
+        "path": "icons/eyedropper.png",
+		"x_offset": 5,
```
Comment (Gymnasiast) [inline resources/g2/sprites.json]: Could you replace the tabs with four spaces each?

**Comment by AaronVanGeffen** [comment]
Thank you for your PR — the icon change was long overdue. However, as I mentioned on Gitter, I think the icon needs slight adjustment for it to fit better in the aesthetics of the existing icons. The edges are too soft, I think. Perhaps make the icon consist of just black and white (or grey) pixels?

Regardless, this change will have to wait until after the release of v0.1.2, as it changes g2.dat.

**Comment by AaronVanGeffen** [comment]
Here's an example to show what I meant before. I've just cranked the brightness down a notch, making the edges rougher to better fit the game's aesthetics.

eyedropper

**Comment by xp4xbox** [comment]
How's this.

**Comment by AaronVanGeffen** [comment]
Could you add a new screenshot, too?

**Comment by xp4xbox** [comment]
untitled

**Pull request merged**